### ISSUE TITLE
Bug: When importing a JSON app configuration query datasources are not found

### ISSUE THREAD

**Issue by swright-eti** [openblocks-dev/openblocks#94]
Steps to reproduce

1. Export an application containing Datasources for REST APIs
2. Import the application into a new instance of OpenBlocks
3. Open one of the queries in the OpenBlocks UI

You will see the Data Source field contains a chunk of JSON. See below. 

You can work around this by just selecting the datasources and setting them, in my case to REST Query. OpenBlocks will then helpfully ask if you want to change all datasources in the application. Just answer yes.  

![image](https://user-images.githubusercontent.com/296522/205740134-2e775b42-627a-4efa-ba59-a20d2a94bfd8.png)

**Comment by neon-balcony** [comment]
Hi @swright-eti ,

Thanks for the post! But I won't consider this issue as bug, actually this is a feature by design. 

Why is that? Because data source are used for making connections to your API or database, and credentials (database username & password, api keys, bearer tokens etc.) for a data source are kept inside a data source's configuration, so it shouldn't be exported in JSON otherwise your credentials might leak. Therefore when you import a JSON into another Openblocks instance/workspace, you need to re-create those data sources again. 

So to sum up, I think it's a case where safety is over ease of use.

**Comment by neon-balcony** [comment]
Hi @swright-eti,

Thanks for the report! We'll fix this in next release.

**Comment by lmx1117** [comment]
Hi swright-eti, this issue is already solved, please check it out, and thanks again for your advice and patience. 😉

**Comment by swright-eti** [comment]
Awesome thanks

**Comment by abulka** [comment]
A similar problem occurs when importing an app which uses modules, the modules are not found - even if I have already imported those modules. It looks like the module `"applicationId"` gets changed on import, so the references from the app to that module can never be resolved and don't link up (they refer to the original module id and of course not to the module id of the imported module which is newly allocated).  Should I raise a separate issue for this?

P.S. Also, imported modules don't appear in the `Home / Modules` area.
